fighters/robot/src/status/special_hi.rs

```diff
@@ -109,34 +109,84 @@ unsafe extern "C" fn special_hi_main_loop(fighter: &mut L2CFighterCommon) -> L2C
     let max_fuel = fighter.get_param_float("param_special_hi", "energy_max_frame");
     let fuel_increment = 2.0; // how much fuel is consumed by the charge per frame
     let min_cost = 20.0; // minimum amount of fuel consumed on use
-    let required_fuel = (fuel_increment * charge_frame).clamp(min_cost, max_fuel);
+    let required_fuel = if fighter.is_situation(*SITUATION_KIND_AIR) { (fuel_increment * charge_frame).clamp(min_cost, max_fuel) } else { ((fuel_increment * 2.0) * charge_frame).clamp(min_cost, max_fuel)};
```

Comment:
instead of duplicating the logic here you should really just multiply fuel_increment before line 112